cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -3471,6 +3474,18 @@ String getInternalServiceHostname(String serviceName)    {
 
     }
 
+    private String getLoggingAppenders(String loggingConfiguration) {
```

Comment:
Can be static?

Reply:
In the other PR it is under `Utils` class so it will be refactored later.